agent/stats/container_test.go

```diff
@@ -125,48 +127,6 @@ func TestContainerStatsCollection(t *testing.T) {
 	}
 }
 
-func TestContainerStatsCollectionReconnection(t *testing.T) {
```

Comment:
why is this test deleted? is the reconnection case tested somewhere else?

Reply:
thanks for catching, that was a mistake